### PR TITLE
Add .travis.yml file and tests

### PR DESCRIPTION
**Pull request by david-martin**

### CODE REVIEW

File: test/server.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ exports.after = function(finish) {
   console.log('global after');
   mbaasApi.db({
     "act": "close"
-  }, function(err) {
+  }, function() {
```
Comment (jasonmadigan) [inline test/server.js]: any reason why we removed the err param?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "0.10"
+sudo: false
+before_install:
+  - npm install -g npm@2.14.9
```
Comment (jasonmadigan) [inline .travis.yml]: small thing (and probably not worth changing), but I think we're using 2.13.5 mostly these days (in fhcap at least)

**Comment by jasonmadigan** [comment]
reviewed - will merge when you get a chance to reply to comments

File: test/server.js (hunk 1 above)
Comment (david-martin) [inline test/server.js]: jshint unused params

File: .travis.yml (hunk 2 above)
Comment (david-martin) [inline .travis.yml]: making this change

**Comment by david-martin** [comment]
@jasonmadigan updated version of npm in this template, and others (see mail for full list)

**Comment by jasonmadigan** [comment]
re-reviewed with latest changes

**Pull request merged**